src/content/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app.mdx

```diff
@@ -15,36 +15,40 @@ redirects:
   - /docs/apm/new-relic-apm/installation-configuration/use-multiple-names-app
 ---
 
-import apmInsightsToggleonandOff from 'images/apm_screenshot-full_insights-toggle-on-and-off.png'
+When working with your APM data in New Relic, you may want to view aggregated, "rolled up" data for an application that spans multiple clusters, environments, or data centers, but also be able to view each of the app instance's data separately. You can do this by setting multiple app names for your APM-monitored apps.
 
-When working with your data in New Relic, you may want to view aggregated data for an application across clusters, environments, or data centers, while at the same time be able to view each of your application instance's data individually.
+## Important notes [#important-notes]
```

Comment:
@zuluecho9 I'm not sure how much the "Important notes" header adds here.

I think it would be good to clarify what someone should do here. For example, is the header "Review this information before you get started" true? Or is this information kind of optional/nice to have?

Do we want someone to read this whole doc before they get started using multiple app names? Maybe it's best to state that at the top of this doc.

Otherwise, I'm inclined to just drop the "Important notes" header and make the h3s here into h2s.